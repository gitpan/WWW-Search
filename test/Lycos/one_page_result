http://www.lycos.com/cgi-bin/pursuit?adv=1&matchmode=or&maxhits=100&query=%22Christie+Abbott%22
