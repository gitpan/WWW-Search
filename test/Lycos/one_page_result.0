<HTML>
<HEAD>
<TITLE>Lycos Search: LSAM IB ISI Intelligent</TITLE>

</HEAD>
<!--begin resultheader.inc-->
<BODY BGCOLOR="#FFFFFF">

<!--TOP LEFT-->
<!-- this is the main table which has only two cells in it.  One for
     the nav bar and one for everything else. -->
<TABLE BORDER=0 CELLSPACING=8 CELLPADDING=0 WIDTH=100%>
<TR>
<!-- navbar cell-->
<CENTER>

<TD ALIGN=MIDDLE VALIGN=TOP WIDTH=134 BGCOLOR="#000000">
<A HREF="http://www.lycos.com">
<IMG SRC="http://www.lycos.com/assist/graphics/new-logo.gif" BORDER=0
WIDTH=125 HEIGHT=48 ALT="Lycos"></A>







<BR>
<a href="http://personal.lycos.com/"><IMG SRC="http://www.lycos.com/assist/graphics/your-guide.gif" height=76 width=125 alt="Search Results" border=0></a>


<!-- GUIDEBAR -->	
<br>
<TABLE WIDTH=112 BORDER=0 CELLSPACING=0 CELLPADDING=0 VALIGN=TOP BGCOLOR="#000000">
<TR VALIGN=TOP>	
<TD VALIGN=TOP COLSPAN=2  BGCOLOR="#0066CC">&#160;<FONT FACE="ARIAL,HELVETICA,MS SANS
SERIF,SANS-SERIF" COLOR="#FFFFFF" SIZE="-1"><B>Web Guides</B></FONT>
</TD></TR>
<TR>

<TD BGCOLOR="#EEEEEE">&nbsp;&nbsp;</TD><TD VALIGN=TOP  BGCOLOR="#EEEEEE" valign="top">


<FONT FACE="helvetica, arial, ms sans serif, sans-serif" COLOR="#000000" SIZE="-2">
&#160;<A HREF="/autos/"><B><FONT color="#000000">Autos</FONT></B></A>
		<BR>
&#160;<A HREF="/business/"><b><FONT color="#000000">Business</FONT></b></a>
		<BR>
&#160;<A HREF="/careers/"><b><FONT color="#000000">Careers</FONT></b></A>
			<BR>
&#160;<A HREF="/computers/"><b><FONT color="#000000">Computers</FONT></b></A>
			<BR>
&#160;<A HREF="/education/"><b><FONT color="#000000">Education</FONT></b></A>
			<BR>
&#160;<A HREF="/electronics/"><b><FONT color="#000000">Electronics</FONT></b></A>
	<BR>
	&#160;<A HREF="/entertainment/"><b><FONT color="#000000">Entertainment</FONT></b></A>
			<BR>
&#160;<A HREF="/fashion/"><b><FONT color="#000000">Fashion</FONT></b></A>
			<BR>
&#160;<A HREF="/games/"><b><FONT color="#000000">Games</FONT></b></A>
			<BR>
&#160;<A HREF="/government/"><b><FONT color="#000000">Government</FONT></b></A>
			<BR>
&#160;<A HREF="/health/"><b><FONT color="#000000">Health</FONT></b></A>
			<BR>
&#160;<A HREF="/homefamily"><b><FONT color="#000000">Home/Family</FONT></b></A>
			<BR>
&#160;<A HREF="/internet/"><b><FONT color="#000000">Internet</FONT></b></A>
			<BR>
&#160;<A HREF="/kids/"><b><FONT color="#000000">Kids</FONT></b></A>
			<BR>
&#160;<A HREF="/money/"><b><FONT color="#000000">Money</FONT></b></A>
	<BR>
&#160;<A HREF="/news/"><b><FONT color="#000000">News</FONT></b></A>
	<BR>
&#160;<A HREF="/people/"><b><FONT color="#000000">People</FONT></b></A>
	<BR>
&#160;<A HREF="/realestate/"><b><FONT color="#000000">Real Estate</FONT></b></A>
	<BR>
&#160;<A HREF="/emarket/shopping/"><b><FONT color="#000000">Shopping</FONT></b></A>
	<BR>
&#160;<A HREF="/space/"><b><FONT color="#000000">Space/Sci-Fi</FONT></b></A>
	<BR>
&#160;<A HREF="/sports/"><b><FONT color="#000000">Sports</FONT></b></A>
	<BR>
&#160;<A HREF="/travel/"><b><FONT color="#000000">Travel</FONT></b></a>	<BR>
&#160;<A HREF="/women/"><b><FONT color="#000000">Women</FONT></b></A><BR></FONT>
	</TD>
</tr>

	</TABLE>

<P>
<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=2 WIDTH=112> 
<TR>
<TD WIDTH=112 BGCOLOR="#0066CC" VALIGN=MIDDLE ALIGN="CENTER">
<FONT FACE="HELVETICA,ARIAL" COLOR="#FFFFFF" SIZE=2>
<B>Pictures&amp;Sounds</B></FONT>
</TD>
</TR>
<TR><TD VALIGN=TOP BGCOLOR="#EEEEEE"><CENTER><FONT FACE="HELVETICA, ARIAL, MS SANS SERIF, SANS-SERIF" COLOR="#000000" SIZE=-2><FONT SIZE=-1><A HREF="http://www.lycos.com/picturethis/">
View Millions<BR>of Images!</a></FONT></CENTER></TD></TR>
</TABLE>

<P>
<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=3 WIDTH=112> 
<TR>
<TD WIDTH=112 BGCOLOR="#0066CC">
<FONT FACE="HELVETICA,ARIAL" COLOR="#EEEEEE" SIZE=2>
<B>Search for</B></FONT>
</TD>
</TR>
<TR>

<TD VALIGN=TOP BGCOLOR="#EEEEEE"> 

<FONT COLOR="#FF0000" SIZE=-1 FACE="helvetica, arial">
<B>LSAM IB ISI Intelligent</B>
<br>
<FONT SIZE=-1 FACE="helvetica, arial">&nbsp;
<A HREF="http://search.dejanews.com/dnquery.xp?site=lycos&amp;query=LSAM%20IB%20ISI%20Intelligent&amp;kr=0">Newsgroups</A></A>
<BR>&nbsp;
<a href="http://www.lycos.com/cgi-bin/pursuit?cat=graphics&amp;query=LSAM%20IB%20ISI%20Intelligent&amp;kr=0">Pictures</a>
<br>&nbsp;
<a href="http://www.lycos.com/cgi-bin/pursuit?cat=bn_results&amp;query=LSAM%20IB%20ISI%20Intelligent&amp;kr=0">Books</A>
<br>&nbsp;
<a href="http://www.lycos.com/cgi-bin/nph-bounce?CDnowSRD|http://www.lycos.com/emarket/shopping/music.html">Music</A>
<br>&nbsp;
<a href="http://www.lycos.com/cgi-bin/pursuit?cat=point&amp;query=LSAM%20IB%20ISI%20Intelligent&amp;kr=0">Site Reviews</A>
<BR></FONT>
</TD>
</TR>
</TABLE>

<P>
<TABLE BORDER=0 BGCOLOR="#EEEEEE" CELLSPACING=0 CELLPADDING=0 WIDTH=112> 
<TR>
<TD WIDTH=112 BGCOLOR="#0066CC" COLSPAN=2 >
<FONT FACE="HELVETICA,ARIAL" COLOR="#FFFFFF" SIZE=2>&nbsp;
<B>More Services</B></FONT></A>
</TD>
</TR>
<TR>

<TD BGCOLOR="#EEEEEE">&nbsp;&nbsp;</TD>
<TD VALIGN=TOP BGCOLOR="#EEEEEE">

<FONT FACE="HELVETICA, ARIAL" SIZE=-1">&#149;&nbsp;
<A HREF="http://www.lycos.com/peoplefind/">
<FONT COLOR="#000000">PeopleFind</FONT></A>
<BR>&#149;&nbsp;
<A HREF="http://www.lycos.com/roadmap.html">
<FONT COLOR="#000000">Road Maps</FONT></A>
<BR>&#149;&nbsp;
<A HREF="http://cityguide.lycos.com">
<FONT COLOR="#000000">CityGuide</FONT></A>
<BR>&#149;&nbsp;
<A HREF="http://point.lycos.com/categories/">
<FONT COLOR="#000000">Site Reviews</FONT></A>
<BR>&#149;&nbsp;
<A HREF="http://www.lycos.com/emarket/shopping/">
<FONT COLOR="#000000">Shopping</FONT></A>
<BR>&#149;&nbsp;
<A HREF="http://www.lycos.com/emarket/travel/">
<FONT COLOR="#000000">Travel</FONT></A>
<BR>&#149;&nbsp;
<A HREF="http://209.143.230.5/cgi-cls/display.exe?lycos+class">
<FONT COLOR="#000000">Classifieds</FONT></A>
<BR>&#149;&nbsp;
<A HREF="http://www.companiesonline.com/">
<FONT COLOR="#000000">Companies</FONT></A>
<BR>&#149;&nbsp;
<A HREF="http://www.lycos.com/cgi-bin/nph-bounce?+sclick|http://sc.sideclick.lycos.com/SideClick/51/navtopicslinks.asp_QUES_MEM=51&amp;SEARCHTYPE=L&amp;SEARCHSTRING=LSAM IB ISI Intelligent"><FONT COLOR="#000000">Related Topics</FONT></A>
</TD>
</TR>
<TR>
<TD WIDTH=112 BGCOLOR="#0066CC" COLSPAN=2 >&nbsp;

<A HREF="http://www.lycos.com/missingkids/"><FONT FACE="HELVETICA,ARIAL" COLOR="#FFFFFF" SIZE=2><B>Missing Kids</B></A>

</TD>
</TR>
</TABLE> 
<BR>


<!-- GUIDEBAR -->	
<br>
<TABLE WIDTH=125 BORDER=0 CELLSPACING=4 CELLPADDING=0 VALIGN=TOP>
<TR VALIGN=TOP><TD VALIGN=TOP BGCOLOR="#0066CC">&#160;&#160;<A HREF="http://www.lycos.com/"><FONT FACE="HELVETICA,ARIAL" COLOR="#FFFFFF" SIZE=2><B>Lycos Home</B></FONT></A></TR>
<TR VALIGN=TOP><TD VALIGN=TOP BGCOLOR="#0066CC">&#160;&#160;<A HREF="http://www.lycos.com/help/"><FONT FACE="HELVETICA,ARIAL" COLOR="#FFFFFF" SIZE=2><B>Help</B></FONT></A></TR>
<TR VALIGN=TOP><TD VALIGN=TOP BGCOLOR="#0066CC">&#160;&#160;<A HREF="http://echomail.lycos.com/em_dir/lycos/lycosmail.nsf/email+form?OpenForm"><FONT FACE="HELVETICA,ARIAL" COLOR="#FFFFFF" SIZE=2><B>Feedback</B></FONT></A></TR>
	</TABLE>
	</TD>

<!---- END GUIDEBAR---->


</CENTER>
</TD>
<!--TOP RIGHT-->
<TD VALIGN=TOP>
<CENTER>

<TABLE BGCOLOR="#000000" WIDTH=100% BORDER=0 CELLPADDING=0 CELLSPACING=0>
<TR><TD>
<TABLE BORDER=0 CELLSPACING=1 WIDTH=100% CELLPADDING=1>
<TD ALIGN=CENTER BGCOLOR="#EEEEEE"><FONT SIZE=-1 FACE="HELVETICA,
ARIAL, SANS SERIF, MS SANS SERIF"><A
HREF="http://personal.lycos.com"><FONT COLOR="#CC0000"><B>Personalize</B></FONT></A>&nbsp;&nbsp;</TD>

<TD ALIGN=CENTER BGCOLOR="#EEEEEE">

<FONT SIZE=-1 FACE="HELVETICA, ARIAL, SANS SERIF, MS SANS SERIF">
<A HREF="http://www.lycosemail.com/member/login.page">Free
Email</A>&nbsp;&nbsp;

</TD>

<TD ALIGN=CENTER BGCOLOR="#EEEEEE">

<FONT SIZE=-1 FACE="HELVETICA, ARIAL, SANS SERIF, MS SANS SERIF">
<A HREF="http://chat.lycos.com">Chat</A>&nbsp;&nbsp;

</TD>


<TD ALIGN=CENTER BGCOLOR="#EEEEEE"><FONT SIZE=-1 FACE="HELVETICA, ARIAL, SANS SERIF, MS SANS SERIF"><A HREF="http://news.lycos.com/headlines/TopNews/default.asp">News</A>&nbsp;&nbsp;</TD>

<TD ALIGN=CENTER BGCOLOR="#EEEEEE"><FONT SIZE=-1 FACE="HELVETICA, ARIAL, SANS SERIF, MS SANS SERIF"><A HREF="http://weather.lycos.com">Weather</A>&nbsp;&nbsp;</TD>

<TD ALIGN=CENTER BGCOLOR="#EEEEEE"><FONT SIZE=-1 FACE="HELVETICA, ARIAL, SANS SERIF, MS SANS SERIF"><A HREF="http://investing.lycos.com/">Stocks</A>
</FONT>
</TD></TR></TABLE>
</TR></TD></TABLE>





<FONT FACE="HELEVETICA,ARIAL,MS SANSSERIF, SANS-SERIF" COLOR="#000000" SIZE=-2>
<BR><a href="/cgi-bin/nph-adclick.exe/CID=0003eae70001ab6000000000/ST=lycos/SV=search/A=.lycos.search_lycos."><img  src="http://ads.bfast.com/ad/serve?siteid=12840&space=1250&size=468x60" alt="Barnes and Noble"  width=468 height=60></a></FONT></CENTER>
<!--/AD-->


<TABLE ALIGN=RIGHT WIDTH=113 BORDER=0>
<TR>
<TD>
<!---STARTRIGHT COLUMN ADS-->


&nbsp;<BR>        
<P>




<TABLE WIDTH=112 BORDER=0 CELLPADDING=1 CELLSPACING=1> 
<TR> 
<TD BGCOLOR="#0066CC" VALIGN=MIDDLE ALIGN=MIDDLE>
<TABLE WIDTH=100% BORDER=0 CELLPADDING=9 CELLSPACING=0> 
<TR>
<TD BGCOLOR="#EEEEEE"VALIGN=MIDDLE ALIGN=MIDDLE>

<FONT SIZE=-2 FACE="helvetica, arial">


<A HREF="http://www.lycos.com/cgi-bin/nph-bounce?BNtest|http://barnesandnoble.bfast.com/booklink/click_QUES_sourceid=16797&amp;ISBN=0399143904"><IMG SRC="http://www.lycos.com/assist/graphics/bandn.gif" width=75 height=26 alt="Barnes and Noble" border=0><BR>Win a Trip: Order Tom Clancy's Rainbow Six</a>
<BR><HR>
<A HREF="http://www.lycos.com/cgi-bin/nph-bounce?CDtest|http://www.lycos.com/kclick/D=lycos/P=/C=cdnow/I=partner/K=default/U=http:%252F%252Fwww.cdnow.com%252Ffrom=bbx:s:lym:sr:"><IMG SRC="http://www.lycos.com/assist/graphics/cdnow2.gif" width=75 height=26 alt="CDnow" border=0></A>
<FONT SIZE=-2 FACE="helvetica, arial"><BR><A
HREF="http://www.lycos.com/cgi-bin/nph-bounce?CDtest|http://www.lycos.com/kclick/D=lycos/P=/C=cdnow/I=partner/K=default/U=http:%252F%252Fwww.cdnow.com%252Ffrom=bbx:s:lym:sr:">Find and Buy Music!</FONT></A>
<BR><HR>

<a href="http://www.lycos.com/cgi-bin/nph-bounce?PTtest|http://www.lycos.com/kclick/D=lycos/P=/C=prevtrav/I=partner/K=default/U=http:%252F%252Fwww.lycos.com%252Femarket%252Ftravel%252Ffullservice.html"><IMG SRC="http://www.lycos.com/assist/graphics/preview.gif" width=75 height=25 alt="Preview Travel" border=0><BR>Plan and Purchase your Travel</a>

<HR>

<A HREF="http://www.lycos.com/cgi-bin/nph-bounce?GTEtest|http://yp.gte.net/lsearch"><IMG SRC="/assist/graphics/gte.gif" width=75 height=26 border=0 alt="GTE Yellow Pages"><BR>Search GTE<BR>Yellow Pages</A>
</FONT>
</A>
</TD> 
</TR>  
</TABLE>
</TD>
</TR>
</TABLE>
<P>
<TABLE WIDTH=112 BORDER=0 CELLPADDING=1 CELLSPACING=1> 
<TR> 
<TD BGCOLOR="#000000" VALIGN=MIDDLE ALIGN=MIDDLE>
<TABLE WIDTH=100% BORDER=0 CELLPADDING=3 CELLSPACING=0> 
<TR> 
<TD BGCOLOR="#CECE9C" VALIGN=MIDDLE ALIGN=MIDDLE>
<B>
<a href="http://www.lycos.com/cgi-bin/nph-bounce?Tritest|http://www.tripod.com">
<FONT SIZE=-1 COLOR="#000000" FACE="helvetica, arial">Tripod</A>
</TD> 
</TR> 
<TR> 
<TD BGCOLOR="#FFFFCE" VALIGN=TOP ALIGN=MIDDLE>
<FONT SIZE=-1 FACE="helvetica, arial">
<A HREF="http://www.lycos.com/cgi-bin/nph-bounce?Tripage|http://homepager.tripod.com/">
<FONT COLOR="#000000"SIZE=-1 FACE="helvetica, arial">
11 MBytes Free!<BR>
Build a New
<BR>Web Page</a>


<P><A HREF="http://www.tripod.com/bin/search/pursuit?cat=tripod&amp;query=LSAM%20IB%20ISI%20Intelligent"><FONT COLOR="#000000"SIZE=-1 FACE="helvetica, arial">Search Personal<BR>Home Pages</A><BR>

</TD> 
</TR> 
</TABLE>
</TD>
</TR>
</TABLE>

</TD>
</TR>
</TABLE><!--ENDRIGHT COLUMN ADS-->
<p>
<!---------------------------------------------------------------------->
<!--SEARCH BOX---------------------------------------------------------->
<!---------------------------------------------------------------------->
<FORM ACTION="http://www.lycos.com/cgi-bin/pursuit" METHOD=GET>
<table cellpadding=0 cellspacing=0 border=0 width=340>
<tr>
 <td valign=middle><FONT SIZE=-1 FACE="helvetica, arial, ms sans serif, sans-serif">You searched for:</FONT></td>
 <td valign=middle><input size=15 name=query value="LSAM IB ISI Intelligent"></td>
 <td valign=middle><INPUT TYPE="image" SRC="/assist/graphics/gogetit_trnsp.gif" height=22 width=72 BORDER=0 align=LEFT></td>
</tr>
<TR><TD>&nbsp;</TD><TD COLSPAN=2 VALIGN=TOP >
<A HREF="http://www.lycos.com/cgi-bin/nph-bounce?SafetyNet|http://personal.lycos.com/safetynet/safetynet.asp">

<FONT SIZE=-2 FACE="helvetica, arial">Surf the Web Safely

</A>
</TD></TR>
</table>
</FORM>
<!---------------------------------------------------------------------->
<!---------------------------------------------------------------------->


 

<FONT SIZE=-2 FACE="helvetica, arial, ms sans serif,sans-serif"></FONT>




<!-- kitix catalog=news results=6 display=1 matchmode=or best_match_filter=1 template=news.exists default=news.does_not_exist cache_id=news; -->









<p>
<!---------------------------------------------------------------------->
<!--SHOPPING BAR-------------------------------------------------------->
<!---------------------------------------------------------------------->
<A NAME="shopping"></A>
<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=0>
<TR>
 <TD VALIGN=TOP ALIGN=LEFT BGCOLOR="#0066CC" WIDTH=333><FONT SIZE=3 FACE="helvetica, arial" COLOR="#FFFFFF"><B>&nbsp;&nbsp;Check These Out&nbsp;&nbsp;</B></FONT></TD>
 <TD VALIGN=TOP BGCOLOR="#FFFFFF">&nbsp;</TD>
</TR>
</TABLE>
<!---------------------------------------------------------------------->
<!----------------------------------------------------------------------> 

<FONT SIZE=-1 FACE="helvetica, arial">
<!-- Modified: $ 1998-07-31 15:15:44 by Master Cylinder $ -->
<A HREF="http://www.lycos.com/cgi-bin/nph-bounce?BNcheck|http://www.lycos.com/cgi-bin/pursuit_QUES_cat=bn_results&amp;query=LSAM%20IB%20ISI%20Intelligent&amp;kr=0">
Find And Buy Books About LSAM IB ISI Intelligent at BarnesandNoble.com</A><BR>
<A HREF="http://www.lycos.com/cgi-bin/nph-bounce?CDcheck|http://www.lycos.com/kclick/D=lycos/P=/C=cdnow/I=partner/K=/U=http:%252F%252Fwww.cdnow.com%252Ffrom=bbx:s:lym:sr:">Click To Buy Music @ CDnow</A>

<BR>

<a href="http://www.lycos.com/cgi-bin/nph-bounce?GTEcheck|http://yp.gte.net/lsearch">Search GTE Yellow Pages for "LSAM IB ISI Intelligent"</a>
<br>

<A Href="http://www.lycos.com/cgi-bin/pursuit?cat=homepage&amp;query=LSAM%20IB%20ISI%20Intelligent&amp;kr=0">Search Personal Home Pages for "LSAM IB ISI Intelligent"</A><BR>
<a
href="http://www.lycos.com/cgi-bin/pursuit?cat=inso&amp;query=LSAM%20IB%20ISI%20Intelligent&amp;matchmode=or&amp;fs=key&amp;kr=0">Search Dictionary for "LSAM IB ISI Intelligent"</A>
<br>

<A Href="http://news.lycos.com/search.asp?cat=news&amp;query=LSAM+IB+ISI+Intelligent">Search News for "LSAM IB ISI Intelligent"</A><BR>

</FONT>
<P>




<!---------------------------------------------------------------------->
<!--WEB PAGES BAR------------------------------------------------------->
<!---------------------------------------------------------------------->
<A NAME="pages"></A>
<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=0>
<TR>
 <TD VALIGN=TOP ALIGN=LEFT BGCOLOR="#0066CC" WIDTH=333><FONT SIZE=3 FACE="helvetica, arial" COLOR="#FFFFFF"><B>&nbsp;&nbsp;Web Pages&nbsp;&nbsp;</B></FONT></TD>
 <TD VALIGN=TOP BGCOLOR="#FFFFFF">&nbsp;</TD>
</TR>
</TABLE>
<!---------------------------------------------------------------------->
<!----------------------------------------------------------------------> 

<FONT SIZE=-1 FACE="helvetica, arial">Previous Page&nbsp;&#124;&nbsp;Next Page&nbsp;
<BR>

<DL><P>
<DT><a href="http://www.isi.edu">www.isi.edu</a>
<DD><a href="http://www.isi.edu/ib/"><B>About IB</B></a><BR>LOGO About Intelligent BW ----------- In today's Internet, information services such as WWW, Gopher, WAIS, etc. are oriented towards a react<BR>http://www.isi.edu/ib/<BR><a href="/cgi-bin/pursuit?cat=lycos&first=1&bq=1&br=1&query=LSAM+IB+ISI+Intelligent&matchmode=or&adv=0&mltq=4247897054&mtemp=main_mlt"><FONT
COLOR="#FF0000" size=-2>More Like This</FONT></a><P> 
<a href="http://www.isi.edu/lsam/ib/"><B>About IB</B></a><BR>LOGO Intelligent Bandwidth ----------- About Intelligent Bandwidth Research Projects People Related Work ----------- LSAM home ISI home Page<BR>http://www.isi.edu/lsam/ib/<BR><a href="/cgi-bin/pursuit?cat=lycos&first=1&bq=1&br=1&query=LSAM+IB+ISI+Intelligent&matchmode=or&adv=0&mltq=3174052885&mtemp=main_mlt"><FONT
COLOR="#FF0000" size=-2>More Like This</FONT></a><P> 
<a href="http://www.isi.edu/ib/plan.html"><B>IB Research Plan</B></a><BR>LOGO IB Research Plan ----------- For the experiments in this research, we will make use of traffic generators that emulate WWW browser requ<BR>http://www.isi.edu/ib/plan.html<BR><a href="/cgi-bin/pursuit?cat=lycos&first=1&bq=1&br=1&query=LSAM+IB+ISI+Intelligent&matchmode=or&adv=0&mltq=2100413399&mtemp=main_mlt"><FONT
COLOR="#FF0000" size=-2>More Like This</FONT></a><P> 
<a href="http://www.isi.edu/ib/discussion.html"><B>IB Discussion</B></a><BR>LOGO IB Discussion ----------- Response time and bandwidth are key resources in this project. Current Internet user interfaces, such as Web <BR>http://www.isi.edu/ib/discussion.html<BR><a href="/cgi-bin/pursuit?cat=lycos&first=1&bq=1&br=1&query=LSAM+IB+ISI+Intelligent&matchmode=or&adv=0&mltq=1026671581&mtemp=main_mlt"><FONT
COLOR="#FF0000" size=-2>More Like This</FONT></a><P> 
<a href="http://www.isi.edu/div7/ib/example.html"><B>IB Research Plan</B></a><BR>LOGO IB Research Plan ----------- For the experiments in this research, we will make use of traffic generators that emulate WWW browser requ<BR>http://www.isi.edu/div7/ib/example.html<BR><a href="/cgi-bin/pursuit?cat=lycos&first=1&bq=1&br=1&query=LSAM+IB+ISI+Intelligent&matchmode=or&adv=0&mltq=2100298554&mtemp=main_mlt"><FONT
COLOR="#FF0000" size=-2>More Like This</FONT></a><P> 
<a href="http://www.isi.edu/lsam/about/"><B>About LSAM</B></a><BR>About LSAM What is LSAM LSAM is developing middleware infrastructure to support scalable distributed information services. This is embodied <BR>http://www.isi.edu/lsam/about/<BR><a href="/cgi-bin/pursuit?cat=lycos&first=1&bq=1&br=1&query=LSAM+IB+ISI+Intelligent&matchmode=or&adv=0&mltq=2100373603&mtemp=main_mlt"><FONT
COLOR="#FF0000" size=-2>More Like This</FONT></a><P> 

<P>
<DT><a href="http://xent.ics.uci.edu">xent.ics.uci.edu</a>
<DD><a href="http://xent.ics.uci.edu/FoRK-archive/jan98/0429.html"><B>FoRK Archive: LSAM: Large-Scale Active Middleware (formerly </B></a><BR>LSAM: Large-Scale Active Middleware (formerly Cities Online) Rohit Khare (rohit@uci.edu) Tue, 03 Feb 1998 00:05:42 -0800 Messages sorted by:<BR>http://xent.ics.uci.edu/FoRK-archive/jan98/0429.html<BR><a href="/cgi-bin/pursuit?cat=lycos&first=1&bq=1&br=1&query=LSAM+IB+ISI+Intelligent&matchmode=or&adv=0&mltq=2077718263&mtemp=main_mlt"><FONT
COLOR="#FF0000" size=-2>More Like This</FONT></a><P> 

</DL>
<NOBR>
<FONT SIZE=-1 FACE="helvetica, arial">
<B> Previous Page&nbsp;&#124;&nbsp;Next Page &nbsp;&#124;&nbsp;Page:&nbsp;<B><FONT SIZE=+1>1</FONT>
</B></B></FONT></NOBR>
<P>
<CENTER>

<!--SEARCH BOX BEGIN-->

<TABLE BGCOLOR="#FFFFFF" WIDTH=440  CELLSPACING=0 CELLPADDING=3 BORDER=0>
<TR><TD VALIGN=TOP"><FONT SIZE=-1 FACE="helvetica, arial, ms sans
serif, sans-serif">You searched for <B>LSAM IB ISI Intelligent</B></FONT></TD></TR></TABLE>

<FORM ACTION="/cgi-bin/pursuit" METHOD=GET>
<TABLE BGCOLOR="#6699FF" WIDTH=440  CELLSPACING=0 CELLPADDING=3 BORDER=0>
<TR>
<TD VALIGN=TOP WIDTH=100% COLSPAN=3>
<INPUT TYPE="hidden" NAME="adv" VALUE="0">	&nbsp;
<FONT SIZE=-1 COLOR=#000000 FACE=HELVETICA,ARIAL>Search&nbsp;
<SELECT NAME=cat>	
<OPTION SELECTED VALUE=lycos>The Web
<OPTION VALUE=point>Web Site Reviews
<OPTION VALUE=homepage>Personal Homepages
<OPTION VALUE=dejanews>Message Boards
<OPTION VALUE=news>Reuters News
<OPTION VALUE=weather>Weather (city)
<OPTION VALUE=cities>Cities
<OPTION VALUE=inso>Dictionary
<OPTION VALUE=stockfind>Stock (symbol)
<OPTION VALUE=cdnow>Music (artist)
<OPTION VALUE=bn>Books
<OPTION VALUE=graphics>Pictures
<OPTION VALUE=sounds>Sounds
<OPTION VALUE=dl>Downloads
<OPTION VALUE=recipedia>Recipes	</SELECT>	&nbsp;for&nbsp;
<SELECT NAME="npl"> 
<OPTION value="matchmode=or&amp;adv=1" >Any of the words 
<OPTION SELECTED value="matchmode=and&amp;adv=1" >All the words	</SELECT></FONT>
</TD>
</TR>
<TR>
<TD>
<FONT SIZE=-1 COLOR=#6699FF FACE=HELVETICA,ARIAL>earch
</TD>
<TD ALIGN=LEFT VALIGN=ABSMIDDLE>
<INPUT TYPE="text" NAME="query" VALUE="" SIZE=35>
</TD>
<TD ALIGN=LEFT VALIGN=ABSMIDDLE>
<INPUT TYPE="image" SRC="/assist/graphics/gogetit_trnsp.gif" height=22 width=72 BORDER=0 align=LEFT>
<NOBR>&nbsp;&nbsp;&nbsp;&nbsp;  
</TD>
</TR>
</TABLE>
<TABLE>
<TR>
<TD COLSPAN=2>
<FONT SIZE=-1 COLOR=#00000 FACE=HELVETICA,ARIAL>
<INPUT TYPE=RADIO NAME=npl1 value="ignore=pq" checked><B>New</B> Search
<INPUT TYPE=RADIO NAME="npl1" value="pq=LSAM+IB+ISI+Intelligent&amp;adv=1" >Search <B>Within These Results</B>
</TD>
</TR>
<TR>
<TD VALIGN=BOTTOM COLSPAN=3>
<TABLE BORDER=0 WIDTH=420 CELLPADDING=0 CELLSPACING=0>
<TR>
<TD ALIGN=LEFT COLSPAN=2>&nbsp;
<A HREF=http://lycospro.lycos.com/lycospro-nojava.html>
<FONT SIZE=-1 COLOR=#000000 FACE=HELVETICA,ARIAL>
<B>

Lycos Pro Advanced Search

</B></FONT></A>
</TD>
<TD ALIGN=RIGHT>

<A HREF="http://www.lycos.com/help/"><FONT SIZE=-1 COLOR=#000000 FACE=HELVETICA,ARIAL><B>Help</B></FONT></A>&nbsp;&nbsp;<A HREF="http://echomail.lycos.com/echomail"><FONT SIZE=-1 COLOR=#000000 FACE=HELVETICA,ARIAL><B>Feedback</FONT></A>&nbsp;&nbsp;</B>

</TD>
</TR>
</TABLE>
</TD>
</TR>
</TABLE>
</FORM></CENTER>
<!--END SEARCH BOX-->
<P>
<!----AD----> 
<FONT FACE="HELEVETICA,ARIAL,MS SANSSERIF, SANS-SERIF" COLOR="#000000" SIZE=-2>
<CENTER><a href="/cgi-bin/nph-adclick.exe/CID=0003eae70001ab6000000000/ST=lycos/SV=search/A=.lycos.search_lycos."><img  src="http://ads.bfast.com/ad/serve?siteid=12840&space=1250&size=468x60" alt="Barnes and Noble"  width=468 height=60></a></CENTER></FONT>
<!--/AD-->
<P>
<CENTER> 
<FONT FACE="HELEVETICA,ARIAL,MS SANSSERIF, SANS-SERIF" COLOR="#000000" SIZE=-3>
<A HREF=/lycosinc/advertising.html>Advertise with Lycos</A>&nbsp;|&nbsp;
<A HREF="http://www.lycos.com/info/">AboutLycos</A>&nbsp;|&nbsp;
<A HREF="http://www.lycos.com/software/safesearch.html">Download Safe Search Software</A>
<BR>
<A HREF="http://www.lycos.com/lycosinc/legal.html">
<FONT COLOR="#000000">Copyright</A>&copy;</FONT> 1998 Lycos,Inc.All Rights Reserved. 
<BR>Lycos
<font size=-3>
<sup>&#174;</sup></font> is a registered trademark of Carnegie Mellon University
<BR>
<A HREF="http://www.lycos.com/lycosinc/legal.html">Terms and conditions</A> </CENTER>
</TD>
</TR>
</TABLE><IMG SRC="http://www.lycos.com/cgi-bin/etcookies.exe"></BODY></HTML>
