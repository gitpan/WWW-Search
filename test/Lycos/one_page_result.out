http://www.isi.edu/lsam/ib/
http://www.isi.edu/ib/plan.html
http://www.isi.edu/ib/discussion.html
http://www.isi.edu/div7/ib/example.html
http://www.isi.edu/lsam/about/
